prow/jobs/busola/busola-web/periodic-performance-tests-web.yaml

```diff
@@ -0,0 +1,15 @@
+# Periodic jobs
+
+# Description- This prow job is for running performance Tests on kyma web application. 
+periodics:
+- interval: 24h
+  agent: kubernetes
+  name: jmeter
+  spec:
+    containers:
+    - image: i331641/kyma-perf:v1
+      imagePullPolicy: IfNotPresent
+      command: [ "/bin/bash", "-c"]
+      args: ["jmeter -n -t  kyma-web-pipeline.jmx -Jkubeconfig={Kubeconfig-file-name} -Jdomain={kyma-domain} -e -l results/log -o results/"]
```

Comment:
Can you provide more information about the requirements you have to meet?

* Where do you need to store the results? You can use `$ARTIFACTS` env variable for that, so the results are stored on GCS.
* Do you need the source code of busola? If so then you have to add busola repo as `extra_refs`.
* How do you provide input files? Are they bundled in the image? You don't plan on dynamically updating them? Maybe it would be better to keep those files in the busola repo, and use extra_refs.

Reply:
Changed the results to $ARTIFACTS